### DISCUSSION TITLE
Including custom CSS and JavaScript in a MyST website

### DISCUSSION THREAD

**Discussion by nthiery** [jupyter-book/mystmd#1414] [General] [answered]
JupyterBook allows for easy [inclusion of custom CSS and javascript](https://jupyterbook.org/en/stable/advanced/html.html#custom-css-or-javascript) in the produced websites. Can the same be achieved with MyST websites?

**Comment by choldgraf** [comment, marked as answer]
Definitely agree with this - I don't believe it's currently possible easily but here are two relevant issues to track and signal boost 

- https://github.com/jupyter-book/myst-theme/issues/437
- https://github.com/jupyter-book/myst-theme/issues/321

**Reply by nthiery** [reply]
Thanks @choldgraf for the quick feedback!
